cinema/urls.py

```diff
@@ -1,5 +1,6 @@
 from django.urls import path, include
 from rest_framework import routers
+from cinema.views import OrderViewSet
```

Comment:
Code Style: Consider grouping the import with the other imports from `cinema.views` using parentheses to improve readability and maintain consistency.